src/main/java/uk/gov/hmcts/reform/pcs/notify/model/NotificationStatus.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.hmcts.reform.pcs.notify.model;
+
+public enum NotificationStatus {
+    CREATED("created"),
+    SENDING("sending"),
+    DELIVERED("delivered"),
+    PERMANENT_FAILURE("permanent-failure"),
+    TEMPORARY_FAILURE("temporary-failure"),
+    TECHNICAL_FAILURE("technical-failure"),
+    SCHEDULE("schedule"),
+    SCHEDULE_PENDING("schedule pending");
```

Comment:
Happy for this to be
```suggestion
    SCHEDULE_PENDING("schedule-pending");
```